python/src/scipp/plot/resampling_model.py

```diff
@@ -125,6 +125,10 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # TODO See #1469. This is a temporary hack to work around the
         # conversion of coords to edges in model.py.
+        new_meta = {name: var.copy() for name, var in self._array.meta.items()}
+        self._array = PlotArray(data=self._array.data,
```

Comment:
I think this is ok?
@SimonHeybrock what do you think?

Reply:
Do you actually have to make copies of all coords? or would creating the `new_meta` dict actually be enough? (and then using `to_bin_centers` below would make a copy where needed)

Reply:
You mean instead of using `array.meta` in `_resample`, we just store and use `new_meta`, keeping `self._array` untouched? Should work I think.

Reply:
Yes this seems to work have removed the copy. Good spot.